newIDE/app/src/UI/CompactToggleButtons/CompactToggleButtons.module.css

```diff
@@ -0,0 +1,42 @@
+.container {
+  display: flex;
+  align-items: center;
+  background-color: var(--theme-text-field-default-background-color);
```

Comment:
I've copied the text field colors, should I create new props from the theme for that button?